Jellyfin.Plugin.KodiSyncQueue/EntryPoints/UserSyncNotification.cs

```diff
@@ -156,8 +155,20 @@ private void SaveUserChanges(List<MediaBrowser.Model.Dto.UserItemDataDto> dtos,
         {
             KodiSyncQueuePlugin.Instance.DbRepo.SetUserInfoSync(dtos, itemRefs, userId);
             List<string> ids = dtos.Select(s => s.ItemId).ToList();
+            var itemCount = ids.Count;
 
-            _logger.LogInformation("\"USERSYNC\" User {UserId}({Username}) posted {NumberOfUpdates} Updates: {Updates}", userId, userName, ids.Count, string.Join(",", ids.ToArray()));
+            if (itemCount > 0)
+            {
+                _logger.LogInformation(
+                    "User Data Sync: User {UserId}({Username}) posted {NumberOfUpdates} updates",
+                    userId,
+                    userName,
+                    itemCount);
+
+                _logger.LogDebug(
+                    "Updated items: {Updates}",
+                    string.Join(",", ids.ToArray()));
```

Comment:
You can just pass the list of IDs

https://github.com/serilog/serilog/wiki/Structured-Data#collections

If this doesn't log the desired outcome you can still remove the `ToArray` call

Reply:
I just used what was already there. Will test and change according to what's actually logged.